lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -231,6 +227,23 @@ def store(payment_method, options = {})
         end
       end
 
+      def customer(post, payment, options)
+        post[:description] = options[:description] if options[:description]
+        post[:expand] = [:sources]
+        post[:email] = options[:email]
+
+        if billing = options[:billing_address] || options[:address]
+          post.merge!(add_address(billing, options))
+        end
+
+        if shipping = options[:shipping_address]
+          post[:shipping] = add_address(shipping, options).except(:email)
+        end
+
+        options = format_idempotency_key(options, 'customer')
+        commit(:post, 'customers', post, options)
```

Comment:
This is interesting that this is a synchronous call to `/customers` within the flow that later calls `/payment_methods/{:payment_method}/attach`. Is there a remote test flow for `store` that confirms both responses are handled well?